### ISSUE TITLE
Lint/EmptyInPattern AllowComments isn't supported

### ISSUE THREAD

**Issue by bkuhlmann** [rubocop/rubocop#9850]
## Expected behavior

Per the [documentation](https://docs.rubocop.org/rubocop/1.16/cops_lint.html#lintemptyinpattern), it appears you can't use `AllowComments` unless the documentation has a typo?

## Actual behavior

When running Rubocop, I see the following output:

![iTerm2-Yurz0JFI](https://user-images.githubusercontent.com/50245/120825191-5b800100-c516-11eb-9fb8-616a2d8456d7.png)

## Steps to reproduce the problem

Use the following Rubocop configuration:

``` yaml
Lint/EmptyInPattern:
  Enabled: true
  AllowComments: false
```

Then run Rubocop on any Ruby file with the above configuration to see the behavior shown above.

## RuboCop version

```
1.16.0 (using Parser 3.0.1.1, rubocop-ast 1.7.0, running on ruby 3.0.1 arm64-darwin20)
  - rubocop-performance 1.11.3
  - rubocop-rake 0.5.1
  - rubocop-rspec 2.3.0
```
